MathGame.destocot/MathGame/Helpers.cs

```diff
@@ -0,0 +1,127 @@
+﻿using MathGame.Models;
+
+namespace MathGame
+{
+    internal class Helpers
+    {
+        internal static List<Game> games = new List<Game>();
+
+        internal static int DisplayProblems(int score, int num1, int num2, string operatorSymbol, Func<int, int, int> operation)
```

Comment:
As this method also hosts the processing of a user's answer as well as displaying the problems, the following may be a more appropriate name:
```suggestion
        internal static int ProcessProblems(int score, int num1, int num2, string operatorSymbol, Func<int, int, int> operation)
```